modules/dreamview/backend/teleop/teleop.cc

```diff
@@ -39,11 +41,36 @@ TeleopService::TeleopService(WebSocketHandler *websocket)
 void TeleopService::Start() {
   // TODO get topic names from proto
   // TODO update proto to get all modems' info combined with rank
-  modem_info_reader_ = node_->CreateReader<ModemInfo>(
-      "/apollo/networks/modems",
-      [this](const std::shared_ptr<ModemInfo> &msg) { UpdateModemInfo(msg); });
-  daemon_cmd_writer_ = node_->CreateWriter<DaemonServiceCmd>(
-      "/apollo/teleop/daemon_service/cmd");
+
+  modem0_info_reader_ = node_->CreateReader<ModemInfo>(
+      "/apollo/teleop/network/modem0",
```

Comment:
move topic strings def to a central place instead of hard-coded it here and there.

Reply:
done [here](https://github.com/ApolloAuto/apollo/pull/9963/commits/075cdc40a1d7c2084e86aed38c980b4e5b433b83)